tools/releaseBuild/azureDevOps/releaseBuild.yml

```diff
@@ -25,7 +25,7 @@ resources:
     type: github
     endpoint: ComplianceGHRepo
     name: PowerShell/compliance
-    ref: master
+    ref: add-mariner-signing
```

Comment:
Need to merge https://github.com/PowerShell/Compliance/pull/47 and change this back to master

Reply:
```suggestion
    ref: master
```